pandas/core/categorical.py

```diff
@@ -1647,14 +1766,17 @@ def mode(self):
 
         import pandas.hashtable as htable
         good = self._codes != -1
-        result = Categorical(sorted(htable.mode_int64(_ensure_int64(self._codes[good]))),
-                             categories=self.categories,ordered=self.ordered, fastpath=True)
+        result = Categorical(
+            sorted(htable.mode_int64(_ensure_int64(self._codes[good]))),
```

Comment:
perhaps put this one in a temporary variable, then the line break at `Categorical(` isn't necessary